Library/Homebrew/extend/os/mac/utils/bottles.rb

```diff
@@ -54,6 +54,11 @@ def find_or_later_tag(tag)
             MacOS::Version.from_symbol(later_tag) <= tag_version
           elsif ARGV.force_bottle?
             true
+          # Allow prerelease versions to act as if all bottles are `_or_later`
+          # so that they don't need to wait for us to bottle everything for the
+          # new release.
+          elsif OS::Mac.prerelease?
```

Comment:
I think we will end up wanting to keep the behavior in place until most bottles are built, which will occur after we've marked Mojave not prerelease (so e.g. the Mojave-is-unsupported messaging stops).